.github/ISSUE_TEMPLATE/03_general_feature.yml

```
@@ -0,0 +1,26 @@
+name: 💡 General feature (not including API proposals)
+description: Add a new feature that does not change the public API surface
```

PRs add new features, issues don't.

```suggestion
description: Discuss a new feature that does not change the public API surface
```

---

😆 

yepp :)